server/core/src/main/scala/sttp/tapir/server/metrics/Metric.scala

```diff
@@ -63,4 +63,25 @@ object MetricLabels {
       }
     )
   )
+
+  /** Default labels for OpenTelemetry-compliant metrics, as recommended here:
+    * https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-server
+    *
+    *   - `http.request.method` - HTTP request method (e.g., GET, POST).
+    *   - `path` - The request path or route template.
+    *   - `http.response.status_code` - HTTP response status code (200, 404, etc.).
+    */
+  lazy val OpenTelemetryAttributes: MetricLabels = MetricLabels(
+    forRequest = List(
+      "http.request.method" -> { case (_, req) => req.method.method },
+      "path" -> { case (ep, _) => ep.showPathTemplate(showQueryParam = None) }
```

Comment:
`url.scheme` is listed as required?

Reply:
same for [error.type](https://opentelemetry.io/docs/specs/semconv/attributes-registry/error/)

Reply:
we can also set [http.route](https://opentelemetry.io/docs/specs/semconv/attributes-registry/http/) by using `Endpoint.showPathTemplate` (using the appropriate formatting). Although this might provide non-trivial overhead to the metric collection